src/main/java/org/itsallcode/junit/sysextensions/SystemErrGuard.java

```diff
@@ -51,6 +51,7 @@ protected Namespace getNamespace()
         return NAMESPACE;
     }
 
+    @SuppressWarnings("java:S106")
```

Comment:
Please add a short comment with a justification for the exclusion. Not everyone knows what `java:S106` means ;)